cypress/integration/Transfer/filter-options-list/index.js

```diff
@@ -0,0 +1,133 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+import { extractOptionFromElement } from '../common'
+
+Given('filtering is enabled', () => {
```

Comment:
Why are there empty Given clauses here?

Reply:
This is due to the nature of cypress and cucumber.
These are background steps so they're true for all scenarios (which makes sense from a feature-file structure perspective).

But we can't assert it's true because the step happens before the step that's loading the correct story. The story already has been prepared to match this scenario, so I kept it as a no-op

Reply:
Ok, I see. In that case it makes more sense to me to add the wording to the step that's loading the story. If the story is providing multiple steps, then I'd reflect that in the title, instead of having empty clauses that don't actually do anything.